integration-sqs-to-lambda-with-batch-item-handling/snippet-data.json

```diff
@@ -0,0 +1,74 @@
+{
+  "title": "Amazon SQS trigger to invoke a Lambda function",
+  "description": "Using an Amazon SQS trigger to invoke a Lambda function",
+  "type": "Integration",
+  "services": ["lambda", "sqs"],
+  "languages": [".NET", "TypeScript"],
+  "tags": [],
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "The Lambda function retrieves a batch of SQS messages from the event parameter that it receives. The function logs the content of each message. When an exception occurs for one or more of the received messages, the handler returns SQSBatchResponse that contains batchItemFailure for failed messages that can be used for message reprocessing or move failed messages to DLQ"
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-snippets/tree/main/integration-sqs-to-lambda-with-batch-item-handling"
+    }
+  },
+  "snippets": [
+    {
+      "title": "Runtimes",
+      "codeTabs": [
+        {
+          "id": ".NET",
+          "title": "Usage Example with .NET:",
+          "description": "Consuming an SQS event with Lambda using .NET.",
+          "snippets": [
+            {
+              "snippetPath": "Function.cs",
```

Comment:
Should this .NET example be here? I can't see the file commited?

Reply:
I updated snippet-data.json file to only include the typescript sample